### PR TITLE
get rid of probin for flame_wave

### PR DESCRIPTION
**Pull request by zingale**
## PR checklist

- [ ] test suite needs to be run on this PR
- [ ] this PR will change answers in the test suite to more than roundoff level
- [ ] all newly-added functions have docstrings as per the coding conventions
- [ ] the `CHANGES` file has been updated, if appropriate
- [ ] if appropriate, this change is described in the docs

### CODE REVIEW

File: Exec/science/flame_wave/inputs_2d.testsuite [hunk 1]
```diff
@@ -90,3 +90,46 @@ amr.derive_plot_vars = ALL
...
+problem.dtemp = 1.2e9
+problem.x_half_max = 2.048e4
+problem.x_half_width = 2048.e0
+
+problem.dx_model = 20.e0
+
+problem.dens_base = 3.43e6
+
+problem.T_star = 1.e8
+problem.T_hi = 2.d8
```
Comment (maxpkatz) [inline Exec/science/flame_wave/inputs_2d.testsuite:106]: d

File: Exec/science/flame_wave/inputs_3d.testsuite [hunk 2]
```diff
@@ -88,3 +88,46 @@ amr.derive_plot_vars = ALL
...
+problem.dtemp = 1.2e9
+problem.x_half_max = 2.048e4
+problem.x_half_width = 2048.e0
+
+problem.dx_model = 20.e0
+
+problem.dens_base = 3.43e6
+
+problem.T_star = 1.e8
+problem.T_hi = 2.d8
```
Comment (maxpkatz) [inline Exec/science/flame_wave/inputs_3d.testsuite:104]: d

File: Exec/science/flame_wave/inputs_3d.testsuite.gpu [hunk 3]
```diff
@@ -88,3 +88,46 @@ amr.derive_plot_vars = ALL
...
+problem.dtemp = 1.2e9
+problem.x_half_max = 2.048e4
+problem.x_half_width = 2048.e0
+
+problem.dx_model = 20.e0
+
+problem.dens_base = 3.43e6
+
+problem.T_star = 1.e8
+problem.T_hi = 2.d8
```
Comment (maxpkatz) [inline Exec/science/flame_wave/inputs_3d.testsuite.gpu:104]: d

Comment (zingale) [reply Exec/science/flame_wave/inputs_3d.testsuite.gpu]: fixed

File: Exec/science/flame_wave/inputs_2d.testsuite (hunk 1 above)
Comment (zingale) [reply Exec/science/flame_wave/inputs_2d.testsuite]: fixed

File: Exec/science/flame_wave/inputs_3d.testsuite (hunk 2 above)
Comment (zingale) [reply Exec/science/flame_wave/inputs_3d.testsuite]: fixed

**Review by maxpkatz — APPROVED** [review]

**Comment by zingale** [comment]
tests: http://groot.astro.sunysb.edu/Castro/test-suite/gfortran/2021-05-24-003/flame_wave.html

**Pull request merged**